include/boost/gil/extension/io/formats/pnm/write.hpp

```diff
@@ -188,7 +188,7 @@ class writer< Device
                           >
                    > buf( src.width() );
 
-        typedef typename View::value_type pixel_t;
+        // typedef typename View::value_type pixel_t;
```

Comment:
Yeah, the commented-out code just below uses these typedefs, so I thought it unwise to remove it, just in case someone decides to un-comment that block.

Reply:
Makes sense, thanks